src/templates/summary/index.ts

```diff
@@ -1,8 +1,15 @@
+import removeMd from 'remove-markdown';
 import { getProposals } from '../../helpers/snapshot';
 import buildMessage from '../builder';
 
 export default async function prepare(params: any) {
   const proposals = await getProposals(params.address);
 
+  Object.keys(proposals).forEach(status => {
```

Comment:
Wouldnt be simpler to do that within getProposals function?

Reply:
As I see it, `getProposals` is like a raw API function to get the proposals from the hub.

By putting the truncation there, we lose the option to use the full body later. We may need the full body later, for emails such as "new proposal".

The need for truncation is coming from the summary email itself, and that logic should belong there, the builder purpose is to fetch and prepare the data

Reply:
We dont need to overwrite the body, could be added as a new param

Reply:
There is already lot of manipulation we are doing on getProposals, for example group them by state, so I dont think its a problem to add the short description there, later we would group by both state and space so it would add more conplexity if we define short description in prepare

Reply:
So `getProposals()` will return a `body`, and `shortBody` params ? I suppose we also need to add an additional params to the function, to customize the length of the truncated body ?

Reply:
Yes, but we dont need additional param to the function. We dont need different length, and if we need it in future we could do it with handlebars